### DISCUSSION TITLE
Please support vanilla JavaScript

### DISCUSSION THREAD

**Discussion by manuthecoder** [tailwindlabs/headlessui#1020] [Ideas]
Hi, 
I don't use React, or Vue either for my project, and I want to use headless UI.
However, I can't find any Vanilla JS Headless UI option.

you really should support this, more people would start using this..

**Comment by georgekettle** [comment]
I'd second this, I'd love this in Vanilla JS. It looks so amazing. I'd use it for every project.

**Comment by mrnebbi** [comment]
Seconded!

**Comment by thomaswilley** [comment]
+1!

**Comment by abdulgidcoder** [comment]
+1
